js/README.md

```diff
@@ -1,17 +1,62 @@
-Jupyter ThreeJS Widget
-======================
+# Jupyter ThreeJS Widget
 
-Interactive Threejs widget for the Jupyter notebook
+[![Interactive demo on Binder][binder-badge]][binder]
+[![Install from PyPI][pypi-badge]][pypi]
+[![Install from conda-forge][cf-badge]][cf]
+[![Reuse from npm][npm-badge]][npm]
+[![Documentation Status][docs-badge]][docs]
+[![Build Status][ci-badge]][ci]
+
+Interactive ThreeJS widgets for the Jupyter Notebook Classic and JupyterLab
+
+![Screencast]
+
+[binder-badge]: https://mybinder.org/badge_logo.svg
+[binder]: https://mybinder.org/v2/gh/jupyter-widgets/pythreejs/HEAD?urlpath=lab%2Ftree%2Fexamples%2FExamples.ipynb
+[pypi-badge]: https://img.shields.io/pypi/v/pythreejs?logo=pypi
+[pypi]: https://pypi.org/project/pythreejs
+[cf-badge]: https://img.shields.io/conda/vn/conda-forge/pythreejs?logo=conda-forge
+[cf]: https://anaconda.org/conda-forge/pythreejs
+[npm-badge]: https://img.shields.io/npm/v/jupyter-threejs?logo=npm
+[npm]: https://www.npmjs.com/package/jupyter-threejs
+[docs-badge]: https://readthedocs.org/projects/pythreejs/badge/?version=stable
+[docs]: https://pythreejs.readthedocs.io/en/stable
+[ci-badge]: https://github.com/jupyter-widgets/pythreejs/actions/workflows/ci.yml/badge.svg
+[ci]: https://github.com/jupyter-widgets/pythreejs/actions/workflows/ci.yml?query=branch%3Amaster
+[widgets]: https://jupyter.org/widgets
+[screencast]: https://raw.githubusercontent.com/jupyter-widgets/pythreejs/master/screencast.gif
 
 The reference Python backend, `pythreejs` is available
-[here](https://github.com/jupyter-widgets/pythreejs), with examples.
+[here](https://github.com/jupyter-widgets/pythreejs), with examples. Install it
+with:
 
-Package Install
----------------
+```bash
+pip install pythreejs
+```
 
-**Prerequisites**
-- [node](http://nodejs.org/)
+or
+
+```bash
+conda install -c conda-forge pythreejs
+```
+
+## Development Install
+
+### Prerequisites
+
+- [nodejs](http://nodejs.org/)
 
 ```bash
 npm install --save jupyter-threejs
 ```
+
+`yarn`, or the JupyterLab-bundled version `jlpm`, may be also be used.
+
+```bash
+yarn add jupyter-threejs
+jlpm add jupyter-threejs
+```
```

Comment:
The `npm/yarn/jlpm add` command is not a dev install prerequisite. I believe the previous bit here was also not very helpful, so it is probably best to delete all of it and just link to the contributing document.

Reply:
cool.

Reply:
Tuned that up, and added the LICENSE so that it gets included in the npm distribution (and all the other places).

Reply:
also, of note: there's no `.npmignore` or `files` in package JSON, so... a _lot_ of stuff gets in there (scripts, src, the linter config, the lab dist tarball, etc). that could probably be tuned up...

Reply:
Good point, I'll have a look at cleaning up the tarball.